layout/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <Import Project="$(MSBuildProjectName).props" />
+  
+  <PropertyGroup>
+    <IsLineupPackage Condition="'$(PackageTargetRuntime)' == ''">true</IsLineupPackage>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
+    <ProjectReference Include="@(Project)" />
+    
+    <!-- Include refs -->
+    <File Include="$(CompileDirectory)*.*">
+      <TargetPath>ref/$(TargetFramework)</TargetPath>
+    </File>
+
+    <FilePackageDependency Include="Microsoft.NETCore.Platforms">
+      <Version>$(PlatformPackageVersion)</Version>
+    </FilePackageDependency>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
+    <NativeFile Include="$(RuntimeDirectory)*.a" />
+    <NativeFile Include="$(RuntimeDirectory)*.so" />
+    <NativeFile Include="$(RuntimeDirectory)*.dylib" />
+    <NativeFile Include="$(RuntimeDirectory)clrcompression.dll" Condition="Exists('$(RuntimeDirectory)clrcompression.dll')" />
```

Comment:
I suppose we need to think about what will be most natural once we switch to vertical build.  Probably just a separate directory for native bits rather than having to list out all modules.  I'll make a change to the buildtools side.